Xamarin.PropertyEditing/Reflection/ReflectionPropertyInfo.cs

```diff
@@ -61,7 +62,7 @@ public virtual void SetValue<T> (object target, T value)
 			this.propertyInfo.SetValue (target, realValue);
 		}
 
-		public virtual T GetValue<T> (object target)
+		public virtual async Task<T> GetValueAsync<T> (object target)
```

Comment:
Same question here on not really being `async`. Keep in mind I don't know this code at all. :-)